script/redirect-cc.cmd

```diff
@@ -1,2 +1,3 @@
 @ECHO OFF
-python ..\..\brave\script\redirect-cc.py %*
+XCOPY /D /Y /Q "..\..\brave\script\redirect-cc.py" "%CD%/redirect.*" 1>nul
```

Comment:
I think it would be better to copy this before the build starts, maybe in brave-browser build.js

Reply:
Addressed by c263257ae366952c82dca6732390df19822d9bd7 and by https://github.com/brave/brave-browser/pull/9765/commits/6f5a1d09c26e5bc592cd0fd749a8a36a5d1ed54c